src/screens/CreateCommunity/index.js

```diff
@@ -97,11 +97,18 @@ const CreateCommunity = () => {
     if (name && name.length >= MINIMUM_NAME_LENGTH && typeFetch === 'available') {
       const response = await submitCommunityName(name);
       if (response.success) {
-        navigation.replace('CreateCommunityCustomize', {
+        navigation.replace('ContactScreen', {
           isCreateCommunity: true,
           topicCommunityId: response.topic_id,
           topicCommunityName: name
         });
+
+        // disabled v4 add images
+        // navigation.replace('CreateCommunityCustomize', {
+        //   isCreateCommunity: true,
+        //   topicCommunityId: response.topic_id,
+        //   topicCommunityName: name
+        // });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Commenting out code is not a good practice for version control. If this code is not needed anymore, it's better to remove it. If it might be needed in the future, consider using feature flags or similar mechanisms to enable/disable features.
```diff
-        // disabled v4 add images
-        // navigation.replace('CreateCommunityCustomize', {
-        //   isCreateCommunity: true,
-        //   topicCommunityId: response.topic_id,
-        //   topicCommunityName: name
-        // });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->